.travis.yml

```diff
@@ -1,9 +1,18 @@
 language: php
-
 php:
-    - 5.3
```

Comment:
Note: removing php 5.3 requires a new major release.

Reply:
What do you mean by *major release*?

Reply:
Thats not true